src/components/controllers/tutorial/style.module.less

```diff
@@ -55,7 +55,6 @@
 			background: var(--color-sidebar-bg);
 			color: var(--color-text);
 			display: flex;
-			flex: 1 1 var(--x, 50%);
```

Comment:
I noticed and removed a fair handful of unused flex rules. None of these elements are ever flex items as far as I can see, so I'm guessing they're just left overs.

Reply:
also me fumbling with Flexbox :P